src/components/SortBlock/SortBlock.tsx

```diff
@@ -0,0 +1,28 @@
+import { ButtonGroup } from '@mui/material';
+import React from 'react';
+import ButtonElem from '../ButtonElem';
+import style from './SortBlock.module.scss';
+
+function SortBlock() {
+  const onClick = (e: React.MouseEvent<HTMLButtonElement>) => {
+    const el = e.target as HTMLButtonElement;
+    const prevEl = document.getElementById('active-sort') as HTMLButtonElement;
```

Comment:
Bad practice to use imperative approach. Don't use DOM directly.
Please create a state to handle the active button and use classnames for comparing current active button from state and your buttons and then apply the active style if needed